float64 x
float64 y
float64 z

float32 roll
float32 pitch
float32 yaw

bool get_aruco
bool not_get_aruco